api_tests/src/test_environment.ts

```diff
@@ -0,0 +1,119 @@
+import { Script } from "vm";
+import { Config } from "@jest/types";
+import { execSync } from "child_process";
+import commander from "commander";
+import { LedgerRunner } from "../lib/ledgers/ledger_runner";
+import { HarnessGlobal } from "../lib/utils";
+import NodeEnvironment from "jest-environment-node";
+import rimraf from "rimraf";
+import * as fs from "fs";
+import { Mutex } from "async-mutex";
+
+// ************************ //
+// Setting global variables //
+// ************************ //
+
+interface ConfigInterface {
+    ledgers: string[];
+    logDir: string;
+}
+
+export class E2ETestEnvironment extends NodeEnvironment {
+    private docblockPragmas: string;
+    private projectRoot: string;
+    private testRoot: string;
+    private logDir: string;
+    private ledgerRunner: LedgerRunner;
+    // @ts-ignore
```

Comment:
Please add a reason for the `@ts-ignore` usage.

Reply:
removed, this was not necessary.